src/Turtle/Prelude.hs

```diff
@@ -195,7 +202,11 @@ import System.Environment (
     lookupEnv,
 #endif
     getEnvironment )
-import System.Directory (getPermissions, readable)
+import qualified System.Directory as Filesystem
+    ( Permissions, getPermissions, setPermissions, emptyPermissions
```

Comment:
Conflict resolution. Some names from `System.Directory` clash with names from `Filesystem.Path.CurrentOS`, and I qualified `System.Directory` as `Filesystem`. Should I qualify it with another name?